packages/plugin-core/src/components/views/PreviewPanel.ts

```diff
@@ -150,13 +151,42 @@ export class PreviewPanel implements PreviewProxy, vscode.Disposable {
   hide(): void {
     this.dispose();
   }
+  lock() {
+    const activeTextEditor = VSCodeUtils.getActiveTextEditor();
+    if (activeTextEditor) {
+      this._lockedEditorFileName = activeTextEditor?.document.fileName;
+      this.sendLockMessage(this._panel!, this.isLocked());
+    } else {
+      vscode.window.showInformationMessage("No active texteditor found");
```

Comment:
Good point, since we do the sanittyCheck in `packages/plugin-core/src/commands/TogglePreviewLock.ts 24` we should not enter this path.

[`7485b1a` (#3293)](https://github.com/dendronhq/dendron/pull/3293/commits/7485b1a7f3e8176005b058039d8fc5d60da73866)